service/src/main/kotlin/fi/espoo/evaka/attendance/StaffAttendanceQueries.kt

```diff
@@ -44,7 +44,12 @@ fun Database.Read.getStaffAttendances(unitId: DaycareId, now: HelsinkiDateTime):
             SELECT jsonb_agg(jsonb_build_object('start', p.start_time, 'end', p.end_time, 'type', p.type))
             FROM staff_attendance_plan p
             WHERE e.id = p.employee_id AND tstzrange(p.start_time, p.end_time) && tstzrange(:rangeStart, :rangeEnd)
-        ), '[]'::jsonb) AS planned_attendances
+        ), '[]'::jsonb) AS planned_attendances,
+        EXISTS(
+            SELECT 1 FROM staff_attendance_realtime osar
+            WHERE osar.employee_id = dacl.employee_id
+              AND NOW() < osar.arrived
```

Comment:
You should probably use `now` from the function parameters for better testability.